datasets/speechocean762/README.md

```diff
@@ -0,0 +1,260 @@
+---
+pretty_name: speechocean762
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+language:
+- en
+license:
+- cc-by-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 100M<n<1B
+source_datasets:
+- original
+task_categories:
+- audio-classification
+task_ids:
+- pronunciation-scoring
+---
+
+# Dataset Card for speechocean762
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://www.openslr.org/101/
+- **Repository:** https://github.com/jimbozhang/speechocean762
+- **Paper:** https://arxiv.org/abs/2104.01378
+- **Leaderboard:** https://paperswithcode.com/dataset/speechocean762
+- **Point of Contact:** [Junbo Zhang](mailto:dr.jimbozhang@gmail.com)
+
+### Dataset Summary
+
+This dataset aims to provide a free public dataset for the pronunciation scoring task.
+
+Five experts made the scores at phoneme-level, word-level, and sentence-level. To avoid subjective bias, each expert scores independently under the same metric.
+
+### Supported Tasks and Leaderboards
+
+-`pronunciation scoring`: This dataset is designed for pronunciation assessment at sentence-level, word-level and phoneme-level.
+
+There is a [leaderboard](https://paperswithcode.com/dataset/speechocean762) for phoneme-level scoring on Papers With Code.
+
+
+
+### Languages
+
+English. Note that the mother tongue of all the speakers is Mandarin. 
+
+## Dataset Structure
+
+### Data Instances
+
+Below is a typical meta data of an instance of the dataset:
+
+```
+"014350017": {    ,
```

Comment:
Can you provide one single example here instead ? This way users know exactly what the content of the dataset looks like.

You can get one using `load_dataset` and `ds["train"][0]`